pyproject.toml

```diff
@@ -73,6 +73,7 @@ respx = "0.20.2"
 tzdata = "2024.1"
 coverage = "^7.4.1"
 databases = {version = "^0.8.0", extras = ["mysql"]}
+python-json-logger = "^2.0.7"

```

Comment:
Is this used anywhere?

edit: I see it exists as a disabled option in the config

Reply:
Yeahh there's a debate that it's not necessary but I think it's a frequent enough use case (structured logging) where I think it's worth having batteries-included